stories/button.stories.tsx

```diff
@@ -18,18 +18,93 @@ const button = storiesOf('Components/Button', module)
         content="Danger"
         className="danger"
       />,
+      <Button
+        callback={action('clicked')}
+        content="Success"
+        className="success"
+      />,
       <Button
         callback={action('clicked')}
         content="Inverted"
         className="inverted"
       />,
+      <Button
+        callback={action('clicked')}
+        content="Squared"
+        className="squared"
+      />,
+      <Button
+        callback={action('clicked')}
+        content="Bottom-Squared"
+        className="bottom-squared"
+      />,
+      <Button
+        callback={action('clicked')}
+        content="Small"
+        className="small"
+      />,
+      <Button
+        callback={action('clicked')}
+        content="Big"
+        className="big"
+      />,
+      <Button
+        callback={action('clicked')}
+        content="Transparent"
+        className="transparent"
+      />,
       <Button
         callback={action('clicked')}
         content="Disabled"
         disabled={true}
       />
     ];
   })
+  .add('with different placement', () => {
+    return [
+      <div className="button-container">
+        <Button
+          callback={action('clicked')}
+          content="Left-Align"
+          className="left-align"
+        />
+        <Button
+          callback={action('clicked')}
+          content="Right-Align"
+          className="right-align"
+        />
+      </div>,
+      <div className="button-container">
+        <Button
+          callback={action('clicked')}
+          content="Top-Align"
+          className="top-align"
+        />
+        <Button
+          callback={action('clicked')}
+          content="Bottom-Align"
+          className="bottom-align"
+        />
+      </div>,
+      <div className="button-container">
+        <Button
+          callback={action('clicked')}
+          content="Navbar"
+          className="navbar-btn"
```

Comment:
I just realized that having a class for buttons in the navigation doesn't make it very reusable or customizable because you're dictating how the buttons should look, and maybe not everyone wants to have the same navigation menu. It's fine but don't think it's very reusable.

Reply:
Not a big deal for now but perhaps moving it back into the registry admin. If that's the use case for this style then it probably belongs there instead of this generic repo.